app/views/case_contacts/form/_contact_types.html.erb

```diff
@@ -1,65 +1,32 @@
-<div id="contact-type-form">
+<div id="contact-type-form" class="d-flex flex-column gap-4">
   <h4 id="contact-type-label"><label>2. Select All Contact Types</label><span class="red-letter"> *</span></h4>
-  <% if controller.controller_name == 'casa_cases' %>
-    <div class="row">
-      <div class="col-sm">
-        <a class="main-btn primary-btn-outline btn-hover btn-sm" id="check_all" onClick="checkContactTypes()">Check All</a>
-        <a class="main-btn primary-btn-outline btn-hover btn-sm" id="uncheck_all" onClick="uncheckContactTypes()">Uncheck All</a>
-      </div>
-    </div>
+  <p>Choose from the available options below by searching or selecting from the dropdown menu.</p>
+  <% last_case = @casa_cases.sort_by(&:created_at).last %>
```

Comment:
seems like there was already a convention of storing some of the view logic state in the partial before, so I stuck with it for speed. Would be happy to extract to a presenter or a decorator in order to encapsulate viewlogic here.